src/components/FormSelectionContainer/FormSelect/FormSelect.tsx

```diff
@@ -1,9 +1,10 @@
 import { Button } from "@govtechsg/tradetrust-ui-components";
-import React, { FunctionComponent, useState, useRef, useEffect } from "react";
+import React, { FunctionComponent, useState, useRef, useEffect, useCallback } from "react";
 import ReactTooltip from "react-tooltip";
 import { useConfigContext } from "../../../common/context/config";
 import { FormTemplate } from "../../../types";
 import { validateDns, getIssuerAddress, getIssuerLocation } from "../../../utils";
+import { checkDID, checkOwnership } from "../../../services/publishing/prechecks";
 
```

Comment:
lets put `prechecks` stuff in a folder on their own?

Reply:
I think i'll move in the DNS stuff later